synapse/cmds/cron.py

```diff
@@ -571,6 +571,9 @@ def _format_timestamp(ts):
                 self.printf(f'                 {incunit:10} {incval:6} {reqdict}')
 
     async def runCmdOpts(self, opts):
+        self.printf('Warning: Cron jobs should now be accessed via storm commands. '
```

Comment:
Can we please use the the warnings library to handle warnings as opposed to printing data?